hosts/web01/configuration.nix

```diff
@@ -6,10 +6,29 @@
     inputs.srvos.nixosModules.mixins-nginx
     inputs.srvos.nixosModules.hardware-hetzner-cloud
     inputs.self.nixosModules.common
+    ./samba.nix
+    ./postgresql.nix
+    ./pgbackrest.nix
+    ./lemmy.nix
   ];
 
   networking.hostName = "web01";
   networking.hostId = "1cfd5aa3";
 
+  # Remove once https://github.com/NixOS/nixpkgs/pull/241651 can be used.
```

Comment:
`xfs` was merged in https://github.com/NixOS/nixpkgs/pull/241526.

Reply:
Great. I had also made a PR but had missed that one.

Reply:
Noticed that your PR has xfs enabled by default and the PR that was merged doesn't, need to enable it in srvos as well. https://github.com/numtide/srvos/pull/183

Reply:
Updated our srvos input so this commit can be dropped. https://github.com/nix-community/infra/pull/660

Reply:
Dropped and rebased.